napari/components/_viewer_key_bindings.py

```diff
@@ -56,18 +55,21 @@ def toggle_ndisplay(viewer: Viewer):
 # ```
 # RuntimeError: wrapped C/C++ object of type CanvasBackendDesktop has been deleted
 # ```
-@register_viewer_action(trans._("Toggle theme."))
+@register_viewer_action(trans._("Toggle current viewer theme."))
 def toggle_theme(viewer: Viewer):
```

Comment:
Maybe annotate as `ViewerModel` not `Viewer` as it looks compatible with `ViewerModel`.

Did the comment above actually?

Reply:
Originally, this was Viewer. Then I made it ViewerModel and it worked. But not for the menu action, which gave a TypeError, so I switched it back to Viewer in this commit:
https://github.com/napari/napari/pull/5143/commits/b26a1fb2dbbbb38323d6f8fc97cfb69632e26ca8

I'm not sure what you mean about the comment? The one abut the RunTimeError?
I haven't seen any issues with tests. 🤷‍♂️

Reply:
With removing the menu changes, I've gone back to ViewerModel here.